downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc

```diff
@@ -9,7 +9,7 @@
 
 .Procedure
 . Log in to {AAPCentralAuth} as an SSO admin user.
-. Under the Configure section on the side navigation bar, click btn:[User Federation].
+. From the  navigation bar, select menu:Configure section[User Federation].
```

Comment:
Looks like there is an extra space between the and navigation.

Reply:
Thanks for catching that! I've made that change now.